flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/FlinkOperatorTest.java

```diff
@@ -74,4 +93,16 @@ public void testConfigurationPassedToJOSDK() {
 
         Assertions.assertThrows(IllegalStateException.class, () -> new FlinkOperator(secondConfig));
     }
+
+    @Test
+    public void testLeaderElectionConfig() {
+        var operatorConfig = new Configuration();
+        operatorConfig.set(KubernetesOperatorConfigOptions.OPERATOR_LEADER_ELECTION_ENABLED, true);
+
+        try {
+            new FlinkOperator(operatorConfig);
+        } catch (IllegalConfigurationException ice) {
+            assertTrue(ice.getMessage().startsWith("Lease name must be defined"));
```

Comment:
Nit: Maybe we can mention either the feature (leader election) or config name. Such case one don't need to dig for conf name just copy-paste.

Reply:
Oh, now see that the feature later mentioned. Maybe adding the key would provide more convenience.

Reply:
Good idea, I will add this